src/main/java/cubicchunks/asm/CubicChunksMixinConfig.java

```diff
@@ -144,5 +153,46 @@ public static String getNicelyFormattedName(String name) {
         }
         return out.toString();
     }
-
+    
+    private void writeConfigToJson(File configFile) throws IOException {
+        JsonWriter writer = new JsonWriter(new FileWriter(configFile));
+        writer.setIndent(" ");
+        writer.beginArray();
+        for (BoolOptions configOption : BoolOptions.values()) {
+            writer.beginObject();
+            writer.name("name");
+            writer.value(configOption.name());
+            writer.name("description");
```

Comment:
Gson should support comments is you enable that feature, could you use that instead and simply store the values in normal way?

something like

```java
/* create comment in whatever way needed */
writer.name(configOption.name());
writer.value(configOption.value);
```